.github/workflows/security.yml

```
@@ -0,0 +1,21 @@
+name: security
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  push:
+    paths:
+      - '**/Cargo.toml'
+      - '**/Cargo.lock'
+
+jobs:
+  audit:
+    runs-on: ubuntu-latest
+    permissions: write-all
+    name: "Audit Dependencies"
```

Perhaps during the transition period we should just be explicit here?

```suggestion
    name: "Audit Rust Dependencies"
```